checkerboard_detector/CMakeLists.txt

```diff
@@ -35,12 +35,6 @@ add_executable(objectdetection_transform_echo src/objectdetection_transform_echo
 target_link_libraries(objectdetection_transform_echo ${catkin_LIBRARIES})
 add_executable(checkerboard_calibration src/checkerboard_calibration.cpp)
 target_link_libraries(checkerboard_calibration ${catkin_LIBRARIES})
-add_dependencies(checkerboard_detector    posedetection_msgs_gencpp)
-add_dependencies(checkerboard_calibration posedetection_msgs_gencpp)
-add_dependencies(objectdetection_transform_echo posedetection_msgs_gencpp)
-add_dependencies(checkerboard_detector    jsk_recognition_msgs_gencpp)
-add_dependencies(checkerboard_calibration jsk_recognition_msgs_gencpp)
-add_dependencies(objectdetection_transform_echo jsk_recognition_msgs_gencpp)
 add_dependencies(checkerboard_detector ${PROJECT_NAME}_gencfg)
```

Comment:
I think this might be problem when we have message source 
c.f. https://github.com/jsk-ros-pkg/jsk_visualization/pull/692/files#r189476048